### PR TITLE
Adding Authentic

### PR DESCRIPTION
**Pull request by davidguttman**

### CODE REVIEW

**Comment by nodiscc** [comment]
@n1trux should we move this to https://github.com/n1trux/awesome-sysadmin#tools-and-web-interfaces also?

**Comment by n1trux** [comment]
Reading the description on its website, I think it's better suited in some node list than in awesome-sysadmin because the non-dev sysadmin doesn't like to write code for services to integrate into something.

File: README.md [hunk 1]
```diff
@@ -434,6 +434,7 @@ See https://github.com/n1trux/awesome-sysadmin#dns
   * [OpenCA OCSPD](https://pki.openca.org/projects/ocspd/) - An [OCSP](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol) Responder.
   * [OpenID/OpenID Connect](http://openid.net/developers/libraries/) - A Simple Identity layer on top of OAuth 2.0.
   * [OSIAM](https://www.osiam.org) - Open Source Identity and Access Management implementing OAuth 2.0 and SCIMv2. ([Source Code](https://github.com/osiam/osiam)) `MIT` `Java`
+  * [Authentic](https://github.com/davidguttman/authentic) - Authentication for microservices using JWT. ([Demo](http://dry.ly/authentic)) `MIT` `node.js`
```
Comment (AndyR207) [inline README.md]: The language should be `Nodejs`

**Comment by AndyR207** [comment]
Closing this because it's been open for over a month with no developments.

**Pull request closed without merging**